src/lang/english.txt

```diff
@@ -1337,10 +1337,12 @@ STR_CONFIG_SETTING_SMALLMAP_LAND_COLOUR_HELPTEXT                :Colour of the t
 STR_CONFIG_SETTING_SMALLMAP_LAND_COLOUR_GREEN                   :Green
 STR_CONFIG_SETTING_SMALLMAP_LAND_COLOUR_DARK_GREEN              :Dark green
 STR_CONFIG_SETTING_SMALLMAP_LAND_COLOUR_VIOLET                  :Violet
-STR_CONFIG_SETTING_REVERSE_SCROLLING                            :Reverse scroll direction: {STRING2}
-STR_CONFIG_SETTING_REVERSE_SCROLLING_HELPTEXT                   :Behaviour when scrolling the map with the right mouse button. When disabled, the mouse moves the camera. When enabled, the mouse moves the map
-STR_CONFIG_SETTING_FREE_SCROLLING                               :Free mouse scrolling: {STRING2}
-STR_CONFIG_SETTING_FREE_SCROLLING_HELPTEXT                      :Behaviour when scrolling the map with the right mouse button. When disabled, the mouse pointer is fixed in position. When enabled, the mouse pointer moves freely. Reverse scroll direction is ignored when enabled
+STR_CONFIG_SETTING_SCROLLMODE                                   :Viewport scroll behaviour: {STRING2}
+STR_CONFIG_SETTING_SCROLLMODE_HELPTEXT                          :Behaviour when scrolling the map
```

Comment:
Maybe the helptext could mention mouse buttons somehow, so people get "RMB" and "LMB".